content/en/blog/hacktoberfest-2024

```diff
@@ -0,0 +1,15 @@
+---
+title: "Hacktoberfest 2024"
+date: 2024-10-07T16:53:13+01:00
+author: Mauro Morales ([Twitter](https://twitter.com/mauromrls)) ([GitHub](https://github.com/mauromorales))
+---
+
+I can't believe it's already October. I've been so busy following up on different events where we've brought some of the Kairos love that I forgot about [Hacktoberfest](https://hacktoberfest.com). Thankfully, the month isn't over so let's get our hands dirty!
+
+Like [last year](https://kairos.io/blog/2023/09/26/hacktoberfest-2023/) Kairos is taking part of this event. This means that you can open PRs in any of our repositories, and we'll make sure to tag them correctly so they count towards your progress (please contact us if this is not the case).
+
+If you'd like to participate, start by looking at the tickets with the [hacktoberfest label](https://github.com/kairos-io/kairos/issues?q=is%3Aopen+is%3Aissue+label%3Ahacktoberfest), but you can hack on any other ticket and we'll be happy to add the tag for you.
+
+This year, we will offer Kairos t-shirts and stickers to the first 10 contributions. So don't wait too long to participate.
```

Comment:
@Itxaka I kinda thought that when we review we can say, this is not enough, like this is a typo, sorry but how about you also reword this page or so. Otherwise, I'm sure I will write 300 lines of requirements and someone will still figure out the one I didn't write. I'm happy to add more info here though, just would also like to keep it open in that sense

Reply:
Maybe something along, `prizes are awarded at kairos discretion and only quality PRs are taken into account (so typos or removing whitespace are counted)`